pkg/verifier/notation/notationrevocationfactory.go

```diff
@@ -35,7 +36,11 @@ var (
 
 // NewCRLHandler returns a new NewCRLHandler instance. Enable cache by default.
 func NewCRLHandler() RevocationFactory {
```

Comment:
Since we have `CreateCRLHandlerFromConfig`, wonder if `NewCRLHandler` is ever used?

Reply:
NewCRLHandler is not used for now. We can remove it.